relayer/relayer/src/trader/mod.rs

```diff
@@ -0,0 +1,303 @@
+use std::{
+    fmt::Debug,
+    str::FromStr,
+    sync::Arc,
+    time::{Duration, SystemTime, UNIX_EPOCH},
+};
+
+use aleph_client::{pallets::system::SystemApi, AccountId, AsConnection, SignedConnectionApi};
+use ethers::providers::Middleware;
+use log::{debug, error, info, warn};
+use thiserror::Error;
+use tokio::{select, sync::broadcast, time::sleep};
+
+use crate::{
+    config::Config,
+    connections::{azero::AzeroConnectionWithSigner, eth::SignedEthConnection},
+    contracts::{AzeroContractError, AzeroEtherInstance, MostInstance, RouterInstance},
+    helpers::left_pad,
+    CircuitBreakerEvent,
+};
+
+// trader component will sell the surplus
```

Comment:
Suggestion: I'm not sure what this comment is meant to be conveying? Is it a description of this whole component? Then make it a moduledoc and perhaps expand on it. Is it a description of these two constants? Then they are inappropriately named - should be `AZERO_SELLOFF_THRESHOLD`, etc. or something like that.

```suggestion
```